src/Components/FunFact.js

```diff
@@ -0,0 +1,113 @@
+import app from '../firebase';
+import { getDatabase, ref, onValue, remove } from 'firebase/database';
+import { useState, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart, faTrashCan } from '@fortawesome/free-regular-svg-icons';
+
+const FunFact = ({ theFact, loveIt }) => {
+  // Set state for 'favorite'
+  const [ favorite, setFavorite ] = useState(false);
+
+  // Make icons functional with enter key
+  const handleKeyDownHeart = (e) => {
+    if(e.keyCode === 13){
+      loveIt(theFact);
+      showPopUp();
+      setFavorite(true);
+    }
+  }
+
+  const handleKeyDownTrash = (e) => {
+    if(e.keyCode === 13){
+      setFavorite(false);
+      //remove from Firebase using key/id
+      const database = getDatabase(app);
+      const dbRef = ref(database);
+      
+      onValue(dbRef, (response) => {
+        const res = response.val();
+        for(let key in res) {
+          if(theFact === res[key]) {
+            const currentKey = key;
+            const database = getDatabase(app);
+            const dbRef = ref(database, `/${currentKey}`);
+            remove(dbRef);
+          }
+        }
+      })
+    }
+  }
+
+  // Function to show app message randomly
+  const showPopUp = () => {
+    // Random chance to determine if app should show message
+    const randomChance = (Math.floor(Math.random() * 100));
+    console.log(randomChance)
```

Comment:
Consider removing this console.log from the production build.

Reply:
Great catch! That was the last bit I was modifying and forgot to remove it.